shapely/geometry/linestring.py

```diff
@@ -56,7 +56,7 @@ def __geo_interface__(self):
             'coordinates': tuple(self.coords)
             }
 
-    def svg(self, scale_factor=1., stroke_color=None):
+    def svg(self, scale_factor=1., stroke_color=None, opacity=None):
```

Comment:
@MejstrikRudolf what would you think about putting the default in the function declaration?

```suggestion
    def svg(self, scale_factor=1., stroke_color=None, opacity=0.8):
```

Currently the opacity is 1, but I think there are advantanges to 0.8, it reveals overlapping shapes. @om-henners (as the creator of this feature) do you see any downsides?

Reply:
Sure, it's better to put the default value to the function declaration. I'm just beginner so I don't have best performence skills yet.
And I need this option for extracting shapes to svg and then putting the svg onto web, so I need for the shapes changeable opacity.

Reply:
@sgillies I think it's a good idea honestly - trying it locally the opacity of 0.8 looks pretty good - I certainly don't see any downsides!